web/vital_records/views.py

```diff
@@ -108,6 +126,21 @@ def form_valid(self, form):
         self.object.save()
         return HttpResponseRedirect(self.get_success_url())
 
+    def get_display_county(self, context):
+        counties = VitalRecordsRequest.COUNTY_CHOICES
+        county_of_birth_id = context["vital_records_request"].county_of_birth
+
+        # Make sure the ID is not blank ("") and ID is in the county options list
+        if county_of_birth_id != "" and county_of_birth_id in dict(counties):
+            return dict(counties)[county_of_birth_id]
+        else:
+            return ""
```

Comment:
Makes sure the `county_of_birth` is not a blank or default blank, and then searches for the ID number (as a string) in a dictionary of the COUNTY_CHOICES list. Not sure if there's a better way to do this.

Reply:
Hmm. Well since you're using a dict, you could convert that once at the beginning and use `.get(key, default)` to make it a little more compact. But nbd.

```python
counties = dict(VitalRecordsRequest.COUNTY_CHOICES)
county_of_birth_id = context["vital_records_request"].county_of_birth
return counties.get(county_of_birth_id, "")
```

Reply:
Ooh, I like this. Will keep this in mind for future refactor, post-deadline. It's an interesting problem b/c there are so many ways to do this.